packages/taco/package.json

```diff
@@ -43,11 +43,13 @@
     "@nucypher/nucypher-core": "*",
     "@nucypher/shared": "workspace:*",
     "ethers": "^5.7.2",
+    "jsonpath": "^1.1.1",
```

Comment:
As @theref noted, this package seems to be no longer maintained. `jsonpath-plus` is in maintenance mode:

> Please note: This project is not currently being actively maintained. We may accept well-documented PRs or some simple updates, but are not looking to make fixes or add new features ourselves.

We may want to switch to `jsonpath-plus` or https://www.npmjs.com/package/nimma

Reply:
I see the same message on the JSONPath-plus GitHub (https://github.com/JSONPath-Plus/JSONPath)

<img width="1039" alt="Screenshot 2024-07-31 at 14 42 17" src="https://github.com/user-attachments/assets/3716c790-6a68-4f93-a888-e6c367cb2c55">

This messaging also appears to be reiterated in this comment on May 15, 2024 https://github.com/JSONPath-Plus/JSONPath/issues/173#issuecomment-2112352687

> As mentioned on the main page of the README, I am not actively maintaining this project, though I am accepting PRs. Feel free to submit a well-documented PR if you find an issue and your question is not answered by the README.

Reply:
In my opinion, since there appears to be no actively maintained jsonpath package, if the original jsonpath package is functioning as expected (demonstrated in this PR) I'm in favor of merging it as-is.  

After all, the only functionality we actually need is the syntactic check and not the evaluation.  If jsonpath supports that but jsonpath-plus does not I don't think we have another way to implement this validator without building our own regular expressions.

Reply:
Due to various issues with [jsonpath](https://www.npmjs.com/package/jsonpath) and jsonpath-plus not functioning as expected, I decided to implement our own JSONPath validator.